src/test01/app.html

```diff
@@ -10,22 +10,24 @@ <h4 class="text-center">Pick a topic, then we'll find a time and place</h4>
 	</div>
 
 	<div ng-if="buttons.length!=5 && buttons.length!=1" class="row extra-bottom-padding-medium" ng-all-same-height target-element="button">
-		<div ng-repeat="item in buttons" class="col-md-6 button-wrapper">
-			<button type="button" class="btn btn-default btn-block">{{ item.text | translate }}</button>
+		<div class="col-md-offset-2 col-md-8">
+			<div ng-repeat="item in buttons" class="col-md-6 button-wrapper">
+				<button type="button" class="btn btn-default btn-block">{{ item.text | translate }}</button>
+			</div>
 		</div>
 	</div>
 
 	<div ng-if="buttons.length==1" class="row extra-bottom-padding-medium">
-		<div ng-repeat="item in buttons" class="col-md-12 button-wrapper">
+		<div ng-repeat="item in buttons" class="col-md-offset-2 col-md-8 button-wrapper">
 			<div class="col-md-offset-3 col-md-6">
 				<button type="button" class="btn btn-default btn-block">{{ item.text | translate }}</button>
 			</div>
 		</div>
 	</div>
 
 	<div ng-if="buttons.length==5" class="row extra-bottom-padding-medium" ng-all-same-height target-element="button">
-		<div ng-repeat="item in buttons">
-			<div class="row">
+		<div class="col-md-offset-2 col-md-8">
+			<div ng-repeat="item in buttons" class="row">
 				<div class="col-md-6 button-wrapper">
 					<button type="button" class="btn btn-default btn-block">{{ item.text | translate }}</button>
 				</div>
```

Comment:
Adjusted buttons to look identical to the photos provided (they were a little wider originally).